### ISSUE TITLE
Pb add-ons

### ISSUE THREAD

**Issue by pommedefrance** [Betterbird/thunderbird-patches#82]
hi
i've got betterbird portable for windows (the last one) but each time i'm starting up betterbird, i have to activate each addons..why they turn off after a restart ? bug ?
thks

**Comment by Betterbird** [comment]
Have you read the release notes, OK, the old ones:
https://www.betterbird.eu/releasenotes/index-previous-91.html
CHANGED - The portable launcher now allows moving the portable "installation", to facilitate, set pref extensions.startupScanScopes to 5 (or 7 when using light weight themes)

Please reopen if that doesn't fix it. We should update our support pages. The portable version is a can of worms.

**Comment by pommedefrance** [comment]
yeah ! thks a lot ;-)

**Comment by Betterbird** [comment]
OK, based on your ticked we've added this text - https://www.betterbird.eu/support/

What's the story with the portable version?

Neither Thunderbird nor Betterbird are designed for portable use. The profile contains files which record native absolute path names, as soon as you move the profile, things break. The Thunderbird team have been wise not to offer a portable version and they are not connected to all the portable stuff that floats around, [PortableApps.com](https://portableapps.com/) being the most sophisticated (but there are others). However, they don't mitigate the native absolute path issue and their portable also creates these files: `C:\Users\<user>\AppData\Local\Thunderbird` and `C:\Users\<user>\AppData\Roaming\Thunderbird`.

By popular demand, Betterbird ships officially as a portable version, but it has the same issues mentioned above. We've written our own simple launcher program, BetterbirdLauncher.exe whose source is [published](https://github.com/Betterbird/thunderbird-patches/tree/main/BetterbirdLauncher). In July 2022 we got a bug report that add-ons didn't work any more when moving or renaming the portable profile. We fixed this by removing the add-on startup cache which also records native absolute path names. With the new version we also get complaints about add-ons or themes not working. The solution to these problems is setting preference `extensions.startupScanScopes` to 5 (or 7 when using light weight themes) in the Config Editor.
